test/simpleTestForGenerator/TGPPL-1.0.txt

```diff
@@ -0,0 +1,362 @@
+    Original Work outside the scope of this License or after its termination
+    shall be subject to the requirements and penalties of copyright or patent
+    law in the appropriate jurisdiction. This section shall survive the
+    termination of this License.
+
+12. **Attorneys' Fees.** In any action to enforce the terms of this License
+    or seeking damages relating thereto, the prevailing party shall be
+    entitled to recover its costs and expenses, including, without
+    limitation, reasonable attorneys' fees and costs incurred in connection
+    with such action, including any appeal of such action. This section shall
+    survive the termination of this License.
+
+13. **Miscellaneous.** If any provision of this License is held to be
+    unenforceable, such provision shall be reformed only to the extent
+    necessary to make it enforceable.
+
+14. **Definition of "You" in This License.** "You" throughout this License,
+    whether in upper or lower case, means an individual or a legal entity
+    exercising rights under, and complying with all of the terms of, this
+    License. For legal entities, "You" includes any entity that controls, is
+    controlled by, or is under common control with you. For purposes of this
+    definition, "control" means (i) the power, direct or indirect, to cause
+    the direction or management of such entity, whether by contract or
+    otherwise, or (ii) ownership of fifty percent (50%) or more of the
+    outstanding shares, or (iii) beneficial ownership of such entity.
+
+15. **Right to Use.** You may use the Original Work in all ways not otherwise
+    restricted or conditioned by this License or by law, and Licensor
+    promises not to interfere with or be responsible for such uses by You.
+
+16. **Modification of This License.** This License is Copyright © 2007 Zooko
+    Wilcox-O'Hearn. Permission is granted to copy, distribute, or communicate
+    this License without modification. Nothing in this License permits You to
+    modify this License as applied to the Original Work or to Derivative
+    Works. However, You may modify the text of this License and copy,
+    distribute or communicate your modified version (the "Modified License")
+    and apply it to other original works of authorship subject to the
+    following conditions: (i) You may not indicate in any way that your
+    Modified License is the "Transitive Grace Period Public Licence" or
+    "TGPPL" and you may not use those names in the name of your Modified
+    License; and (ii) You must replace the notice specified in the first
+    paragraph above with the notice "Licensed under <insert your license name
+    here>" or with a notice of your own that is not confusingly similar to
+    the notice in this License.
+=======================================================
+Transitive Grace Period Public Licence ("TGPPL") v. 1.0
+=======================================================
+

```

Comment:
Why is the license text here twice?

Reply:
because apparently I hit paste twice, which I caught in the xml file, but forgot that the text file probably had that too and didn't scroll down enough to notice.... fixed now!